r/tests/testthat/test-dplyr-aggregate.R

```diff
@@ -188,8 +194,32 @@ test_that("Filter and aggregate", {
       filter(int > 5) %>%
       group_by(some_grouping) %>%
       summarize(total = sum(int, na.rm = TRUE)) %>%
-      arrange(some_grouping) %>%
       collect(),
     tbl
   )
 })
+
+test_that("Expressions on aggregations", {
+  # This is what it effectively is
+  expect_dplyr_equal(
+    input %>%
+      group_by(some_grouping) %>%
+      summarize(
+        any = any(lgl),
+        all = all(lgl)
+      ) %>%
+      collect() %>%
```

Comment:
More accurate representation of what this effectively is
```suggestion
      compute() %>%
```